src/unix.rs

```diff
@@ -613,6 +628,7 @@ pub fn set_thread_priority_and_policy(
 
                 // Normal priority threads adjust relative priority through niceness.
                 set_errno(0);
+                #[cfg(not(target_os = "vxworks"))]
```

Comment:
setpriority is not available in VxWorks. But due to the cfg in if condition control never comes here. Thus these were the minimal changes to avoid build errors. I'm not sure if this is the best way though. The best would be to cfg away this code for vxworks, macos and ios.